example/simple_testing/src/lib.rs

```diff
@@ -0,0 +1,137 @@
+// #[diplomat::bridge]
+// mod ffi {
+
+fn test_rust_fn(f: impl Fn(i32) -> i32) {
```

Comment:
nit: Diplomat doesn't really do standalone functions (yet). I'd recommend writing this to look more like a static function on some type so that it's easier to follow along